README.md

```diff
@@ -1,6 +1,8 @@
 # teal.gallery
 
-<img align="right" width="100" height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/thumbs/teal.png">
+<img align="right" width=auto height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/teal.png" style="margin-left:10px;">
+<img align="right" width=auto height="100" src="https://raw.githubusercontent.com/insightsengineering/hex-stickers/main/PNG/teal.gallery.png">
```

Comment:
can we add `href` to redirect to appropriate packages? Now it just opens a new tab with logo

Reply:
agree with this comment - let's add a `href` to the `teal.gallery` package

Reply:
good idea, will do.

Reply:
done, added href to both logos.